tasks/stboot.yml

```diff
@@ -1,7 +1,9 @@
 version: '3'
 
 vars:
+  STBOOT_HOSTCONFIG_EFIVAR: STHostConfig-f401f2c1-b005-4be0-8cee-f2e5945bcbe7
```

Comment:
Did we fully decide on that already? I mean it's a sane default in my opinion, but I just choose that name at random.

Reply:
I guess. :smile: since we now pin all repos, it should not be hard to change it in the future.

Reply:
True